torch/_dynamo/symbolic_convert.py

```diff
@@ -4174,14 +4193,15 @@ def log_graph_break(
             user_stack = torch._guards.TracingContext.extract_stack()
 
         try:
-            frame_loc = (user_stack[-1].filename, user_stack[-1].lineno)
+            lineno = user_stack[-1].lineno
+            frame_loc = (user_stack[-1].filename, lineno if lineno is not None else 0)
```

Comment:
Is it possible to refactor the lineno/frame_loc computation to one place? This is kinda repeated in `_get_frame_loc_chain`.